package.json

```diff
@@ -17,19 +17,19 @@
     "grunt-eslint": "^22.0.0",
     "grunt-mocha-nyc": "^1.0.3",
     "grunt-mocha-test": "0.13.3",
-    "mocha": "^6.2.0",
-    "nyc": "^14.1.1",
+    "mocha": "^7.0.1",
+    "nyc": "^15.0.0",
     "should": "13.2.3",
     "supertest": "4.0.2"
   },
   "scripts": {
     "test": "grunt --verbose"
   },
   "dependencies": {
-    "argon2": "0.24.0",
-    "async": "2.6.3",
+    "argon2": "0.26.0",
+    "async": "3.2.0",
     "basic-auth": "2.0.1",
-    "bluebird": "3.5.5",
+    "bluebird": "3.7.2",
```

Comment:
Oh nice, you have an `allSettled` now :)
https://github.com/petkaantonov/bluebird/releases/tag/v3.7.0

Reply:
Interesting, but it is not in the docs :D
http://bluebirdjs.com/docs/api-reference.html

Reply:
> Updated 06 Nov 2018 

Those docs don't see a lot of action :-/